### PR TITLE
Crud delete

### PR DESCRIPTION
**Pull request by dlopez16**
both routes work properly

Routes
- change the api route endpoint to be /deleteVariant and /deleteProduct

controllers:
-minors changes so the formula could read the id
- forgot to delete all my comments from the delete variant controller :/

### CODE REVIEW

**Review by Mosi-Rivera — APPROVED** [review]
T

**Review by Mosi-Rivera — CHANGES_REQUESTED** [review]
Resuelve los markers de conflicto.

File: controllers/product_controllers.js [hunk 1]
```diff
@@ -165,6 +166,28 @@ module.exports.addCrudProduct = async (req, res, next) => {
 module.exports.updateCrudVariant = async (req, res, next) => {
     try {
 
+<<<<<<< HEAD
```
Comment (Mosi-Rivera) [inline controllers/product_controllers.js:169]: Conflict marker

File: controllers/product_controllers.js [hunk 2]
```diff
@@ -207,17 +231,41 @@ module.exports.updateCrudVariant = async (req, res, next) => {
...
+        // const variant = await Variant.findOne({ _id })
+        res.status(200).json({ message: `variant with id: ${_id} successfully deleted!` });
     } catch (error) {
         res.sendStatus(500) && next(error);
     }
 }
 
 module.exports.addCrudVariant = async (req, res, next) => {
     try {
+<<<<<<< HEAD
```
Comment (Mosi-Rivera) [inline controllers/product_controllers.js:263]: Conflict marker.

File: controllers/product_controllers.js [hunk 3]
```diff
@@ -227,7 +275,12 @@ module.exports.addCrudVariant = async (req, res, next) => {
             "stock.L.stock": lStock,
             "stock.XL.stock": xlStock,
             color,
+<<<<<<< HEAD
```
Comment (Mosi-Rivera) [inline controllers/product_controllers.js:278]: conflict marker

Comment (dlopez16) [reply controllers/product_controllers.js]: hice otro push, tengo que hacer otro pull request, por que no se si ya se refleja en este pull

**Review by Mosi-Rivera — APPROVED** [review]

**Pull request merged**